CMakeLists.txt

```diff
@@ -590,17 +591,16 @@ if(CIRCT_SLANG_FRONTEND_ENABLED)
     # statically link slang into the CIRCTImportVerilog library, but seems to be
     # harder than it ought to be.
     set_property(
-      GLOBAL APPEND PROPERTY CIRCT_EXPORTS slang_slang unordered_dense fmt)
+      GLOBAL APPEND PROPERTY CIRCT_EXPORTS slang_slang fmt)
     
     # Disable the installation of headers coming from third-party libraries. We
     # won't use those APIs directly. Just make them static libraries for the sake
     # of running slang normally.
     set_target_properties(fmt PROPERTIES PUBLIC_HEADER "")
-    set_target_properties(unordered_dense PROPERTIES PUBLIC_HEADER "")
 
-    install(TARGETS slang_slang unordered_dense fmt EXPORT CIRCTTargets)
+    install(TARGETS slang_slang fmt EXPORT CIRCTTargets)
   else()
-    find_package(slang 3.0 REQUIRED)
+    find_package(slang 7.0 REQUIRED)
```

Comment:
TODO: Ask `slang` to mint a release so that this version includes everything that we need, including https://github.com/MikePopoloski/slang/pull/1164.